static/js/impala/suggestions.js

```diff
@@ -130,12 +130,14 @@ $.fn.searchSuggestions = function($results, processCallback, searchType) {
             $results.filter('.visible').removeClass('visible');
             return;
         }
+        var urlVal = encodeURIComponent($self.val());
```

Comment:
The `'` doesn't cause a problem in the url and ends-up as %27 in the resulting URL when inserted. Whilst it seems somewhat overkill I think escaping for the HTML context is still worth doing.

http://jsbin.com/huxadul/edit?js,console,output